RecoEgamma/EgammaElectronProducers/plugins/SealModule.cc

```diff
@@ -19,6 +19,13 @@
 #include "GEDGsfElectronProducer.h"
 #include "GEDGsfElectronFinalizer.h"
 
+#include "LowPtGsfElectronCoreProducer.h"
```

Comment:
Fresh from the new guidelines "Do not split plugins into header and source files" and "SealPlugins.cc or any special files that just define plugins, except for template instantiations, are discouraged" [1].

Probably that's also good for you, because you don't have to perpetually switch from `.h` to `.cc` files with your editor! At least I like this new guideline and here we have a good opportunity to act upon it :)

[1] https://github.com/cms-sw/cms-sw.github.io/blob/a58bf0a9db495d83d9bfc35898d6a5be84d79263/cms_coding_rules.md

Reply:
I've moved the ```DEFINE_FWK_MODULE``` macros out of SealModule.cc and into the .cc files themselves. Is this sufficient?

Reply:
@guitargeek @bainbrid 
The guidelines you are pointing at say "Do not split plugins into header and source files" but also "If you do split them, keep the header files in the .../plugins directory", which is exactly what this PR is doing.
Moreover, moving all macros out of SealModule.cc is certainly welcome, but it could be also postponed to a separated pull request: we are already a bit late with the implementation of this low pt photons reco, and I would stick on it before investing time in other, unrelated optimizations.

Reply:
I've already moved the macros out of SealModule.cc, [here](
https://github.com/cms-sw/cmssw/pull/25696/commits/0faf94565002d6e8cd43b72dd29ce87f4281163e).

I'll leave it at that for now, unless somebody tells me to do otherwise.